src/Reflex/Dom/Internal.hs

```diff
@@ -162,6 +162,9 @@ instance ( MonadRef m, Ref m ~ Ref IO, MonadRef h, Ref h ~ Ref IO --TODO: Should
 --  runWidget :: (Monad m, IsNode n, Reflex t) => n -> Widget t m a -> m (a, Event t (m ()))
   getRunWidget = return runWidget
 
+  getQuitWidget = return $ do wv <- askWebView
+                              liftIO $ quitWebView wv
+
```

Comment:
There's always a HasWebView (WidgetHost m), so can't we just implement this as a top-level function?  If we put it in the class, all the other instances will need to implement it.